demo/src/MemoryModels.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { Button, Card, CardContent, TextField, Grid } from "@mui/material";
+
+interface MemoryModel {
+    isClass: boolean;
+    name: string;
+    id: string | number | null;
+    value: any;
+    stack_frame: boolean;
+    show_indexes: boolean;
+    style?: any[] | null;
+}
+
+function MemoryModels(props) {
```

Comment:
I'm not a big fan of this name. This component is really the user input component, right? Try picking a name that reflects that role